app/views/users/_form.html.erb

```diff
@@ -1,32 +1,28 @@
-<%= form_with(model: user, url: signup_path, class:"mx-1 mx-md-4")  do |form| %>
+<%= form_with(model: user)  do |form| %>
   <%= render 'shared/errors', model: user, model_name: 'user' %>
-  <div class="d-flex flex-row align-items-center mb-4">
-    <i class="fas fa-user fa-lg me-3 fa-fw"></i>
-    <div class="form-outline flex-fill mb-0">
-      <%= form.text_field :first_name, class:"form-control" %>
-      <%= form.label :first_name, class:"form-label"%>
-    </div>
+  <div class="mb-3">
+    <%= form.label :first_name, class:"form-label"%>
+    <%= form.text_field :first_name, class:"form-control" %>
   </div>
-  <div class="d-flex flex-row align-items-center mb-4">
-    <i class="fas fa-user fa-lg me-3 fa-fw"></i>
-    <div class="form-outline flex-fill mb-0">
-      <%= form.text_field :last_name, class:"form-control" %>
-      <%= form.label :last_name, class:"form-label"%>
-    </div>
+  <div class="mb-3">
+    <%= form.label :last_name, class:"form-label"%>
+    <%= form.text_field :last_name, class:"form-control" %>
   </div>
-  <div class="d-flex flex-row align-items-center mb-4">
-    <i class="fas fa-user fa-lg me-3 fa-fw"></i>
-    <div class="form-outline flex-fill mb-0">
-      <%= form.email_field :email, class:"form-control" %>
-      <%= form.label :email, class:"form-label"%>
-    </div>
+  <div class="mb-3">
+    <%= form.label :email, class:"form-label"%>
+    <%= form.email_field :email, class:"form-control" %>
   </div>
-  <div class="d-flex flex-row align-items-center mb-4">
-    <i class="fas fa-user fa-lg me-3 fa-fw"></i>
-    <div class="form-outline flex-fill mb-0">
-      <%= form.password_field :password, class:"form-control" %>
-      <%= form.label :password, class:"form-label"%>
-    </div>
+  <div class="mb-3">
+    <%= form.label :role, class:"form-label"%>
+    <%= form.select :role, User.roles.to_a.map {|r| [r[0].capitalize, r[0]]}, {}, {class: "form-select"} %>
+  </div>
+  <div class="mb-3">
+    <%= form.label :password, class:"form-label"%>
+    <%= form.password_field :password, class:"form-control" %>
+  </div>
+  <div class="d-flex flex-row align-items-center mb-5">
+    <%= form.check_box :show_password, onchange: "showPassword()", class: "form-check-input me-2" %>
+    <%= form.label :show_password, 'Show password', class: 'form-check-label' %>
   </div>
   <div class="d-flex justify-content-center mx-4 mb-3 mb-lg-4">
     <%= form.submit class:"btn btn-primary btn-lg" %>
```

Comment:
@rishabh-vinsol 
in users edit view
pls remove show this user link, as you have removed show route
Also `Back to users` is not visible below
Add that at the top where you are showing edit user or new user
like -> `Users/New User` -> here `User` will be url for users index page

Make this change wherever the links are not visible.

Reply:
Done